package.json

```diff
@@ -81,11 +82,10 @@
     "@types/vcf": "^2.0.3",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
-    "@vue/devtools": "^5.3.4",
```

Comment:
Why's this being removed?

Reply:
Breaks yarn completely.

Remove as discussed.